torch/distributed/tensor/parallel/style.py

```diff
@@ -166,6 +166,15 @@ def _apply(self, module: nn.Module, device_mesh: DeviceMesh) -> nn.Module:
             ),
         )
 
+    def __repr__(self) -> str:
+        tmpstr = self.__class__.__name__ + "("
+        tmpstr += f"src_data_rank={self.src_data_rank}, "
```

Comment:
I think we should remove `src_data_rank` from `__repr__` (including all the following changes). Python's `__repr__` ideally can be used to recreate the object using eval()

Reply:
Thanks for your review! Have made changes and please take a look, thanks!